src/modules/weather/weather.js

```diff
@@ -291,10 +295,14 @@ export class Weather {
 
     async _refreshWeatherContainer() {
         try {
+            this._spinner = new Spinner();
+            this._spinner.showSpinner('body');
```

Comment:
we display spinner in a body and what about the page content? is it moved? you dont need to pass body since its a default value